src/devices/Vl53L0X/Information.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Vl53L0X
+{
+    /// <summary>
+    /// Store the information regarding the sensor
+    /// </summary>
+    public class Information
+    {
+        /// <summary>
+        /// Module ID
+        /// </summary>
+        public byte ModuleId { get; set; }
+        /// <summary>
+        /// The revision number
+        /// </summary>
+        public Version Revision { get; set; }
+        /// <summary>
+        /// The product ID
+        /// </summary>
+        public string ProductId { get; set; }
+        /// <summary>
+        /// Raw measurement of the signal rate fixed point 400 micrometers
+        /// </summary>
+        public UInt32 SignalRateMeasFixed1104_400_Micrometers { get; set; }
+        /// <summary>
+        /// Raw measurement of the distance measurement fixed point 400 micrometers
+        /// </summary>
+        public UInt32 DistMeasFixed1104_400_Micrometers { get; set; }
```

Comment:
`UInt32` -> uint

Reply:
I really prefer UInt32 as it's always super clear. But I'll replace everywhere.